runtime/pendulum/src/xcm_config.rs

```diff
@@ -262,6 +252,54 @@ impl ShouldExecute for DenyReserveTransferToRelayChain {
 	}
 }
 
+// We will allow for BRZ location from moonbeam
+pub struct AutomationPalletConfigPendulum;
+
+impl AutomationPalletConfig for AutomationPalletConfigPendulum {
+	fn matches_asset(asset: &MultiAsset) -> Option<u128> {
+		let expected_multiloc = moonbeam::BRZ_location();
+
+		match asset {
+			MultiAsset { id: AssetId::Concrete(loc), fun: Fungibility::Fungible(amount_deposited) } if loc == &expected_multiloc =>
+				return Some(*amount_deposited),
+			_ => return None,
+		}
+	}
+
+	// TODO modify with automation's pallet instance
+	fn matches_beneficiary(beneficiary_location: &MultiLocation) -> Option<(u8, [u8;32])> {
```

Comment:
Maybe we should create an extra struct for the return type? It's not obvious why we return a tuple `(u8, [u8;32])` here but if we group it under some struct with a descriptive name it would be easier to understand.